src/app/work-item/work-item-detail/work-item-comment/work-item-comment.component.html

```diff
@@ -3,12 +3,12 @@
         <!-- Refactoring needed to toogle view -->
         <div class="col-md-12 col-sm-12 col-xs-12" *ngIf="true">
             <div *ngIf="loggedIn" class="">
-                <textarea rows="2" type="text" id="wi-comment-add-comment" class="form-control width100" [(ngModel)]="comment.attributes.body"
-                    [style.height]="descHeight" [style.resize]="descResize" (keydown.enter)="createComment($event)" (keyup.enter)="preventDef($event)"
-                    placeholder="Add a comment here...">
-                </textarea>
+                <p almEditable id="wi-comment-add-comment" class="form-control placeholder width100"
+                    (focus)="resetCommentDraft($event)" (blur)="resetCommentDraft($event)"
```

Comment:
@nimishamukherjee can you verify?
I have already had fixed in in previous push, and rechecked again. Doesn't happen for me.